### DISCUSSION TITLE
How to turn off lint rules?

### DISCUSSION THREAD

**Discussion by scottmessinger** [rome/tools#3675] [Q&A] [answered]
I feel like an idiot, but I can't find out how to do this anywhere. How do I turn off a specific rule in the`rome.json` file? For instance, how do I turn off `lint/style/useBlockStatements`?

**Comment by sebmck** [comment, marked as answer]
https://docs.rome.tools/linter/#disable-a-lint-rule

**Reply by scottmessinger** [reply]
That was embarrassing 🤦‍♂️ No idea how I missed that. Thanks!!!
